react/components/atoms/input/input.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { withKnobs, boolean, text } from "@storybook/addon-knobs";
+
+import { Input } from "./input";
+
+storiesOf("Components/Atoms/Input", module)
+    .addDecorator(withKnobs)
+    .add("Input", () => {
+        const header = text("Header", "Name");
+        const value = text("Value", undefined);
+        const hasBorder = boolean("Has Border", true);

```

Comment:
only noticed it know, but maybe rename this variable and prop to `showBorder` (like the icon `showIcon` and the standard we are using in components-vue)